modules/redisearch/redisearch.ts

```diff
@@ -6,10 +6,13 @@ import {
     FTAggregateParameters, FTConfig, FTCreateParameters, FTFieldOptions, FTFieldType, FTIndexType, FTInfo, FTSchemaField,
     FTSearchParameters, FTSpellCheck, FTSpellCheckResponse, FTSugAddParameters, FTSugGetParameters
 } from './redisearch.types';
+import { RedisearchHelpers } from './redisearch.helpers';
 
 export class Redisearch extends Module {
 
     private searchCommander = new SearchCommander();
+    private helpers = new RedisearchHelpers();
```

Comment:
Call it searchHelpers and add the same variable to the redis-modules (Redis all in one) class. Else it might break 🤔